tests/appsec/test_asm_standalone.py

```diff
@@ -698,9 +698,14 @@ def test_telemetry_sca_enabled_propagated(self):
         assert cfg_appsec_enabled.get("value") == outcome_value
 
     def setup_app_dependencies_loaded(self):
-        weblog.get("/load_dependency")
+        self.r = weblog.get("/load_dependency")
 
     def test_app_dependencies_loaded(self):
+        # test standalone is enabled and dropping traces
+        for data, _trace, span in interfaces.library.get_spans(request=self.r):
+            assert span["metrics"]["_sampling_priority_v1"] == 0
+            assert span["metrics"]["_dd.apm.enabled"] == 0
```

Comment:
what do you think @vpellan?

Reply:
I have included the check in both tests

Reply:
In Ruby, when the sampler decides to not sample a trace, it sets it to MANUAL_REJECT. So `span["metrics"]["_sampling_priority_v1"]` will be equal to -1 (MANUAL_REJECT) instead of 0 (AUTO_REJECT). Maybe it is an error from our side (we still pass all the standalone ASM threats system-tests though) but if not, we should replace this line with:
`assert span["metrics"]["_sampling_priority_v1"] <= 0`. Otherwise I think it's good !

Reply:
In node 0 is used for automatic resolutions and -1 when `manual.drop` has been found or a rule has dropped the trace. But is OK for me to change the assert!